FitNesseRoot/properties

```diff
@@ -6,4 +6,4 @@ FilesAttributeUpdate=applied
 SymLinkPropertyFormatUpdate=applied
 VirtualWikiDeprecationUpdate=applied
 RecentChangesAttributeUpdate=applied
-WhereUsedAttributeUpdate=applied
+WhereUsedAttributeUpdate=applied
```

Comment:
We still have some change here.

Reply:
Looks like there is an extra carriage return in master. I readded it and pushed back up.